### DISCUSSION TITLE
Lua not found by script, despite being in PATH

### DISCUSSION THREAD

**Discussion by Kyllingene** [atom/atom#22166] [Q&A] [answered]
I am on a Windows 10 pc, and have both the full Lua install and all the required packages for `script` to run Lua. I also have the Lua folder added to PATH. However, when I try to run a .lua file from within Atom, it says it's "not a recognized command", *unless* I set the profile to run from the Lua folder. Is there any reason why Atom fails to access lua.exe?

**Comment by UziTech** [comment, marked as answer]
Did you restart your computer after adding lua to the PATH?

**Reply by Kyllingene** [reply]
Ah, a fool I am! No, I did not (a relative noob to PATH, so I forgot that step). Thanks (and sorry) for your time!

**Reply by ThatXliner** [reply]
Wait, you need to restart your computer after you modified `$PATH` on windows?

**Reply by UziTech** [reply]
No restarting Atom should be enough. I don't know if the Atom window or Atom's main process is what caches the env variables. Just to be safe might as well restart the computer.
